### PR TITLE
Generate fake test results

### PR DESCRIPTION
**Pull request by fejta**
/assign @michelle192837 @munnerz 

Add some more detailed action data.

### CODE REVIEW

File: experiment/resultstore/main.go [hunk 1]
```diff
@@ -286,30 +395,195 @@ func setup(account, tok, invID string) error {
 			},
 		},
 	})
-
-	print("cct", ct, err)
+	if err != nil {
+		return fmt.Errorf("create configured target: %v", err)
+	}
+	print("cct", ct)
```
Comment (michelle192837) [inline experiment/resultstore/main.go]: Can you clarify some of the 1-3 letter variables/log lines in this file? Some of them are quite far from their definition, or hard to read/remember :(

Comment (fejta) [reply experiment/resultstore/main.go]: I promise I'll clean this up and use better variable names (often with multiple syllables!) once I'm refactoring it for others to use.

**Pull request merged**